profiles/car.lua

```diff
@@ -62,19 +62,22 @@ local profile = {
     'motor_vehicle',
     'vehicle',
     'permissive',
-    'designated',
-    'destination',
-    'hov' -- we might filter hov out later depending on the avoid settings or add penalties
+    'designated'
   },
 
   access_tag_blacklist = Set {
     'no',
-    'private',
     'agricultural',
     'forestry',
     'emergency',
-    'psv',
-    'delivery'
+    'psv'
```

Comment:
This is a good point. As someone who has definitely driven onto roads I wasn't supposed to be on, routing out of these is probably a better way to handle them, rather than pretending they don't exist in the road network.